city/schemas.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str | None = None
```

Comment:
The type hint `str | None` uses the new union operator introduced in Python 3.10. If you need to support older Python versions, consider using `Optional[str]` from the `typing` module.